src/components/Select/Select.test.tsx

```diff
@@ -51,42 +50,152 @@ describe("Select", () => {
   it("Renders an aria-label", () => {
     wrapper = Enzyme.mount(
       <Select
-        isRequired={false}
+        labelText="Select Label"
         onChange={changeCallback}
         onBlur={blurCallback}
-        ariaLabel="arialabel"
+        showLabel={false}
         name="test3"
       >
         <option aria-selected={true}>test1</option>
         <option aria-selected={false}>test2</option>
       </Select>
     );
-    expect(wrapper.find("select").props()["aria-label"]).to.equal("arialabel");
+    expect(wrapper.find("select").prop("aria-label")).to.equal("Select Label");
   });
 
-  it("Renders aria-labelledby for labelId and helperTextId", () => {
+  it("Renders an aria-label based on deprecated prop", () => {
     wrapper = Enzyme.mount(
       <Select
-        isRequired={false}
+        labelText="Select Label"
+        ariaLabel="Aria Label"
+        onChange={changeCallback}
+        onBlur={blurCallback}
+        showLabel={false}
+        name="test3"
+      >
+        <option aria-selected={true}>test1</option>
+        <option aria-selected={false}>test2</option>
+      </Select>
+    );
+    expect(wrapper.find("select").prop("aria-label")).to.equal("Aria Label");
+  });
+
+  it("Renders aria-describedby", () => {
```

Comment:
What makes it render an `aria-describedby`? Is it the `helperText` prop? Add that to this test `it` description, 
"Renders aria-describedby because a ____ prop was passed".

Reply:
Good suggestion.  Done.